app/script/cryptography/CryptographyRepository.coffee

```diff
@@ -404,7 +404,7 @@ class z.cryptography.CryptographyRepository
       else
         payload = @_construct_payload @current_client().id
         use_local_sessions = true
-        @logger.log @logger.levels.INFO, 'Encrypt message using local sessions'
+        @logger.log @logger.levels.INFO, "Encrypt '#{generic_message.content}' message using local sessions"
       return @get_sessions user_client_map, use_local_sessions
```

Comment:
In this case `generic_message.content` is the type and not the textual content of the message. :smiley: